jetbrains-ultimate/it/software/aws/toolkits/jetbrains/services/lambda/nodejs/NodeJsLocalLambdaRunConfigurationIntegrationTest.kt

```diff
@@ -51,6 +47,7 @@ class NodeJsLocalLambdaRunConfigurationIntegrationTest(private val runtime: Runt
     private val mockId = "MockCredsId"
     private val mockCreds = AwsBasicCredentials.create("Access", "ItsASecret")
     private val fileContents =
+        // language=JS
```

Comment:
nit: below you use `@Language("JSON")` what's the difference between that form and this form?

Reply:
Both work, we should probably be consistent